examples/remote_control_cozmo.py

```diff
@@ -58,7 +88,11 @@ def create_default_image(image_width, image_height, do_gradient=False):
 remote_control_cozmo = None
 _default_camera_image = create_default_image(320, 240)
 _is_mouse_look_enabled_by_default = False
-_display_debug_annotations = True
+
+DEBUG_ANNOTATIONS_DISABLED = 0
```

Comment:
I would expect these 3 constants to be at the top of the script near the imports instead.